src/aks-preview/azcli-aks-live-test/clone_repo.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
do we still need to clone the repo?

Reply:
we need azure-cli repo to perform following tests, which is neither included in base image nor auto-checked by pipeline(checkout azure-cli-extensions only)

Reply:
we can also check another repro in the same pipeline? I mean from the yaml definition instead of the script.

Reply:
need further investigation, resolve for now